pkg/configuration/constants.go

```diff
@@ -16,5 +16,6 @@ const (
 	WORKFLOW_USE_STDIO             string = "wflstdio"
 	RAW_CMD_ARGS                   string = "raw_cmd_args"
 	WEB_APP_URL                    string = "internal_snyk_app"
-	OAUTH_AUTH_ENABLED             string = "internal_snyk_oauth_enabled"
+	// feature flag to enable the new oAuth user flow
+	OAUTH_AUTH_FLOW_ENABLED string = "internal_snyk_oauth_enabled"
```

Comment:
Suggestion: Since we are already renaming this, should we add `FF_` as a prefix?

Reply:
done

Reply:
I guess I need to change the feature flag name then in LS, too?